### PR TITLE
Support yaml

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: src/j1cli.js [hunk 1]
```diff
@@ -203,28 +212,40 @@ async function mutateRelationships(j1Client, relationships, update) {
 }
 
 async function mutateAlertRules(j1Client, rules, update) {
-  const promises = [];
+  const created = [];
+  const updated = [];
+  const skipped = [];
```
Comment (fomentia) [inline src/j1cli.js]: Are you using these arrays anywhere? I see where you're pushing into them, but I don't see where you're reading them.

Comment (erkangz) [reply src/j1cli.js]: yes, in the future...

Comment (fomentia) [reply src/j1cli.js]: ok

Comment (erkangz) [reply src/j1cli.js]: one more time, pls.  the future is here

Comment (fomentia) [reply src/j1cli.js]: Ah very nice!

**Pull request merged**